docs/supported-technologies.asciidoc

```diff
@@ -47,6 +47,10 @@ We support automatic instrumentation for the following web frameworks.
 |2.x
 |1.0
 
+|ASP.NET (.NET Framework) on IIS
+|3.5 and higher (IIS 7.0 and higher)
```

Comment:
Is `3.5` here the .NET Framework version? If so, was that tested? 🤔  I think the new package would only work from `4.6.1` and higher, this is what we define in the `Elastic.Apm.AspNetFullFramework.csproj`: 
`<TargetFrameworkVersion>v4.6.1</TargetFrameworkVersion>`. I guess you can't add that to a `3.5` project. But maybe I'm wrong... just making sure `3.5` is ok here.

Reply:
10x - Good catch. I've changed it to `4.6.1`.